CHANGELOG.md

```diff
@@ -5,7 +5,10 @@
 - **[NEW]** Add `StepperFullScreen` component
 - **[NEW]** Add `DatePickerOverlay` component
 - **[NEW]** Add `DatePickerFullScreen` component
+- **[NEW]** Add `AutoCompleteOverlay` component
+- **[NEW]** Add `AutoCompleteSection` component
 - **[NEW]** Add `CalendarIcon` component
+- **[NEW]** Add `ItineraryLocationIcon` component
```

Comment:
You may use the `Bullet` component instead 
https://blablacar.github.io/ui-library/?path=/story/widgets-bullet--basic

Reply:
I didn't knew this component, it is exactly what I was looking for! Thanks